libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -483,15 +474,10 @@ private void resetAnalyticsPolicy() {
         e.commit();
     }
 
-    private static ScheduledFuture createPublishHandler() {
-        final ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
```

Comment:
It also worked out well that we can eliminate the `JobIntentService` _and_ this `ScheduledExecutorService` with Android background tasks.